quinn-h3/src/server.rs

```diff
@@ -220,25 +222,69 @@ pub struct Sender {
 }
 
 impl Sender {
-    pub fn send_response<T: Into<Body>>(
-        self,
-        response: Response<T>,
-    ) -> Result<SendResponse, Error> {
-        SendResponse::new(response, self.send, self.stream_id, self.conn)
+    pub fn response<T>(self, response: Response<T>) -> RespBuilder<T> {
+        RespBuilder {
+            response,
+            sender: self,
+            trailers: None,
+        }
     }
+}
 
-    pub fn send_response_trailers<T: Into<Body>>(
-        self,
-        response: Response<T>,
-        trailer: HeaderMap,
-    ) -> Result<SendResponse, Error> {
-        SendResponse::with_trailers(
-            response,
-            Some(trailer),
-            self.send,
-            self.stream_id,
-            self.conn,
-        )
+pub struct RespBuilder<T> {
```

Comment:
Let's just call it `ResponseBuilder`?

Reply:
I wasn't sure about this, this `struct` is always used with `http::RequestBuiler`. But using an abreviation to solve namespace "problem" is maybe not that clean.

Reply:
Renamed.